src/components/Jumpstart.tsx

```diff
@@ -24,8 +51,123 @@ const Template: React.FC = () => {
         <title>Write Title Here</title>
         <meta name="description" content="Write a description for SEO here" />
       </Helmet>
-      <Box my={4}>
-        <Paper className={classes.example} />
+      <Hero
+        subtitle="Our brand new full-stack development program for first time developers."
+        title="Jumpstart"
+        SvgNode={JumpstartSvg}
+      />
+
+      <Grid
+        container
+        justify="space-between"
+        alignItems="flex-start"
+        direction="row"
+        className={classes.section}
+        spacing={6}
```

Comment:
6 seems like a lot on my browser, the apply page which also uses a card-esque layout does spacing={3}, maybe try that instead

Reply:
I was on a large monitor, let me try that